src/notification/streamlabs.ts

```diff
@@ -0,0 +1,42 @@
+import {Link, Store} from '../store/model';
+import {Print, logger} from '../logger';
+import {config} from '../config';
+import {URLSearchParams} from 'url';
+import fetch from 'node-fetch';
+
+const {streamlabs} = config.notifications;
+let requestParams: URLSearchParams;
+
+if (streamlabs.accessToken) {
```

Comment:
```suggestion
if (streamlabs.accessToken && streamlabs.type) {
```

Since `type` is required, I think we should put this here as well.